flocker/control/test/test_httpapi.py

```diff
@@ -242,6 +242,76 @@ def test_container_name_collision_different_node(self):
         """
         return self._container_name_collision_test(self.NODE_A, self.NODE_B)
 
+    def test_create_container_with_environment(self):
+        """
+        A valid API request to create a container including environment
+        variables results in an updated configuration.
+        """
+        saving = self.persistence_service.save(Deployment(
```

Comment:
I think I asked this before, but what's the point of this initial step? Seems to me that the configuration should be clean /empty  at the beginning of each test.
And remember, that users are allowed to request a new container on a node that has not (as yet) been seen.

I think this can be removed, but if not, then add an explanatory comment.